src/Categories/Functor/Properties.agda

```diff
@@ -29,17 +28,13 @@ Contravariant : ∀ (C : Category o ℓ e) (D : Category o′ ℓ′ e′) → S
 Contravariant C D = Functor (Category.op C) D
 
 Faithful : Functor C D → Set _
-Faithful {C = C} {D = D} F = ∀ {X Y} → (f g : C [ X , Y ]) → D [ F₁ f ≈ F₁ g ] → C [ f ≈ g ]
+-- Faithful {C = C} {D = D} F = ∀ {X Y} → (f g : C [ X , Y ]) → D [ F₁ f ≈ F₁ g ] → C [ f ≈ g ]
```

Comment:
Commented out code

Reply:
indeed. But this also changes the definition of `Faithful` as `Injective` is different. Same with `Surjective` below.

I think that `StrictlyInjective` (and ditto for `Surjective`) might be more appropriate.